src/components/InsertExpense.js

```diff
@@ -0,0 +1,298 @@
+
+
+var newDate = new Date();
+var moment = require('moment');
+var dateIn = moment(newDate);
+var formatedDate=dateIn.format("YYYY-MM-DD");
+
+//Adding js styles
+const useStyles = makeStyles( (theme) => (
+  {
+  container: {
+      display: 'flex',     
+      flexWrap: 'wrap',
+ 
+  },
+  datepickerx:{
+  
+      width: "35%",
+      marginLeft:"20%",
+      underline: {
+          "&&&:before": {
+          color: "white"
+          },
+          "&&:after": {
+          color: "white"
+          }
+          },
+          '&& .MuiInput-root:hover::before': {
+              borderColor: 'white',
+          },
+          '& .MuiInput-input':{ color: "white"},
+         
+          color: "white",
+          textEmphasisColor: "white",
+          '& .MuiInput-underline:before': {
+          borderBottomColor: "white",
+          '& .MuiInput-underline:after': {
+              borderBottomColor: 'white',
+          },
+          '& .MuiInput-underline:before': {
+              borderBottomColor: 'white',
+          },
+          multilineColor:{
+              color:'white'
+          },
+          '& label.Mui-focused': {
+              color: 'white',
+          },
+          '& label': {
+              color: 'white',
+          },
+          '&:hover fieldset': {
+              borderColor: 'white',
+          },
+          '&.Mui-focused fieldset': {
+              borderColor: 'white',
+          },
+          
+          
+          },
+      },
+  Button:{
+      marginLeft: "1%",
+      background: 'grey',
+      border: 0,
+      borderRadius: 3,
+      color: "white",
+      width:"90%",
+      padding:"3%",
+      marginTop: "7%",
+      textTransform: 'none',
+      backgroundColor: '#37364b',
+      borderColor: '#007bff',
+      // borderRadius: 10,
+      '&:hover': {
+      backgroundColor: 'black',
+      borderColor: '#0062cc',
+      },
+      '&:active': {
+      boxShadow: 'none',
+      backgroundColor: 'black',
+      borderColor: '#005cbf',
+      
+      },
+      '&:focus': {
+      boxShadow: 'black',
+      },
+      
+      
+  },
+  textField: {
+      marginLeft:"1%",
+      marginRight: "1%",
+      width: "90%",
+      paddingTop: "1%",
+      marginTop: "4%",
+     
+      '& .MuiInput-input':{ color: "white"},
+      multilineColor:{
+      color:'white'
+      },
+      '& label.Mui-focused': {
+      color: 'white',
+      },
+      '& label': {
+      color: 'white',
+      },
+      '& .MuiInput-underline:after': {
+      borderBottomColor: 'white',
+      },
+      '& .MuiInput-underline:before': {
+      borderBottomColor: 'white',
+      },
+      
+      '&:hover fieldset': {
+      borderColor: 'white',
+      },
+      '&.Mui-focused fieldset': {
+      borderColor: 'white',
+      },
+      '&& .MuiInput-root:hover::before': {
+      borderColor: 'white',
+      } 
+      },
+  dense: {
+      marginTop: 19,
+  },
+  menu: {
+      width: 200,
+  },
+  }));
```

Comment:
@Sruthyparvathy Follow hyphen delimited string naming convention to name css class name